src/http/modules/ngx_http_ssl_module.c

```diff
@@ -788,9 +788,18 @@ ngx_http_ssl_merge_srv_conf(ngx_conf_t *cf, void *parent, void *child)
     if (conf->verify) {
 
         if (conf->client_certificate.len == 0 && conf->verify != 3) {
-            ngx_log_error(NGX_LOG_EMERG, cf->log, 0,
-                          "no ssl_client_certificate for ssl_verify_client");
-            return NGX_CONF_ERROR;
+            if (conf->protocols & (NGX_SSL_SSLv3|NGX_SSL_TLSv1))
+            {
+                ngx_log_error(NGX_LOG_EMERG, cf->log, 0,
+                            "no ssl_client_certificate for ssl_verify_client");
```

Comment:
style: continued arguments should indent under the first argument

Reply:
I will address it in suggested commit and will follow the same in future. Thanks.